addons/docs/vue/index.js

```diff
@@ -1 +0,0 @@
-module.exports = require('../dist/frameworks/common/index');
```

Comment:
@shilman can you tell me who uses these files?

Reply:
These are legacy files that should've been removed in 6.0 but we'll need to remove them in 7.0.